implementations/micrometer-registry-signalfx/src/main/java/io/micrometer/signalfx/SignalFxConfig.java

```diff
@@ -42,6 +42,15 @@ default String accessToken() {
         return getSecret(this, "accessToken").required().get();
     }
 
+    /**
+     * Returns {@code true} if the signalfx registry should emit histogram buckets.
+     *
+     * @return {@code true} if the signalfx registry should emit histogram buckets.
+     */
+    default Boolean fixHistogramBucketsType() {
+        return getBoolean(this, "reportHistogramBuckets").orElse(false);
+    }
```

Comment:
Based on the implementation, it seems this does not publish "percentile histograms" but a "minimum histogram" containing the defined Service Level Objective (SLO) boundaries.

Can we match the name of the key, the name of the method, the javadoc and the variables in the registry + tests to this?

```suggestion
    /**
     * Wether to publish a minimum a histogram containing your defined service level objective (SLO) boundaries.
     *
     * @return {@code true} if the SignalFx registry should emit SLO buckets, {@code false} otherwise.
     */
    default Boolean publishSlos() {
        return getBoolean(this, "publishSlos").orElse(false);
    }
```